src/controllers/DoctorLoadedPatientController.java

```diff
@@ -54,13 +57,10 @@ public LinkedHashMap<String, Command> AllCommands() {
         LinkedHashMap<String, Command> commands = new LinkedHashMap<>();
         commands.put("unload patient", Back(previousController));
         commands.put("back", Back(previousController));
```

Comment:
It might be better to put commands related to the doctors actions first, and then the commands for changing state after.